tests/README.rst

```diff
@@ -0,0 +1 @@
+Tests can be found in the subdirectories of dwave_networkx/dwave_networkx.
```

Comment:
I'm not sure if this should be a readme or a comment in the requirements file.., or if there's a better way to do this.

Reply:
This is fine, IMO.